ldap/resource_ldap_group.go

```diff
@@ -83,6 +95,26 @@ func resourceLDAPGroupRead(d *schema.ResourceData, m interface{}) error {
 	return err
 }
 
+func resourceLDAPGroupUpdate(d *schema.ResourceData, m interface{}) error {
+	client := m.(*goldap.Client)
+	dn := fmt.Sprintf("CN=%s,%s", d.Get("name").(string), d.Get("ou").(string))
+
+	if d.HasChange("description") {
```

Comment:
I'm not sure to understand why we need to use HasChange here, the ldap update request should be idempotent. It will be simplified as:
```golang
func resourceLDAPGroupUpdate(d *schema.ResourceData, m interface{}) error {
	client := m.(*goldap.Client)
	dn := fmt.Sprintf("CN=%s,%s", d.Get("name").(string), d.Get("ou").(string))

	if err := client.UpdateGroup(dn, d.Get("name").(string), d.Get("description").(string)); err != nil {
		return err
	}

	return resourceLDAPGroupRead(d, m)
}
```
What do you think?

Reply:
Humm. Definitely ! Fixed in https://github.com/Ouest-France/terraform-provider-ldap/pull/6/commits/254065d74efc774bb623ce348734e2274b2290d0

🤦‍♂️